Description: Fix documentation wrt. to which dhclient.conf wicd (re)generates
Author: Axel Beckert <abe@debian.org>
Bug-Debian: https://bugs.debian.org/773809
Forwarded: not-yet

Index: wicd/in/man=wicd.8.in
===================================================================
--- wicd.orig/in/man=wicd.8.in	2017-01-16 04:11:33.605226283 +0100
+++ wicd/in/man=wicd.8.in	2017-01-22 04:57:34.007529938 +0100
@@ -120,7 +120,7 @@
 
 .B %ETC%dhclient.conf.template
 .br
-This is used to replace /etc/dhclient.conf during Wicd activity,
+This is used to generate /var/lib/wicd/dhclient.conf during Wicd activity,
 if you're using dhclient(1) as DHCP client. See dhclient.conf(5)
 for more information.
 
